floaters/src/main/java/com/mdg/droiders/floaters/ExpandedWindow.java

```diff
@@ -122,6 +131,7 @@ public boolean onTouch(View v, MotionEvent event) {
                                     }
                                     mListener.clickHappened();
                                 }
+                                closeButtonLayout.setVisibility(View.GONE);
```

Comment:
\[Checkstyle\] WARNING: 'block' child have incorrect indentation level 32, expected level should be one of the following: 24, 26, 28\.